src/yz_fuse.erl

```diff
@@ -26,14 +26,22 @@
 -export([create/1, check/1, check_all_fuses_ok/1, melt/1, remove/1, reset/1]).
 
 %% helpers
--export([fuse_context/0]).
+-export([fuse_context/0, encode_fuse_name/2, decode_fuse_name/1,
+         fuse_name_for_index/1]).
```

Comment:
Will do, @fadushin.

On a side note, the Eiffel language had a feature where you could export a function to specific callers. I never saw much utility in it, but it strikes me that functions exported to tests only would be a perfect fit.